packages/block-library/src/cover/editor.scss

```diff
@@ -47,10 +47,6 @@
 		margin: 0;
 	}
 
-	.block-editor-block-list__layout {
```

Comment:
Removing editor specific code is great, let's make sure nothing is regressing though